.yarnrc

```diff
@@ -0,0 +1 @@
+save-prefix ""
```

Comment:
Can you comment on this? Also, the file is missing a newline.

Reply:
This was with usage of `@latest` tag and omitting the `^` before the version. I will remove since we use exact versions now.